.github/workflows/build.msvc.windows.workflow.yml

```diff
@@ -139,7 +140,7 @@ jobs:
       shell: cmd
       run: |
         echo Creating directory to prevent failure: vcpkg\downloads
-        md vcpkg\downloads
+        if not exist vcpkg\downloads md vcpkg\downloads
```

Comment:
This makes sense and prevents setting %ERRORLEVEL%. One thing I also found is this - not sure if worth considering too: https://stackoverflow.com/a/4165472

Reply:
I'm a bit confused...do you just mean to log whether or not the folder exists already?